src/components/editor/inputs/InputLang.jsx

```diff
@@ -33,16 +33,16 @@ const InputLang = (props) => {
   const selectLanguage = (selected) => {
     if (selected.length === 1) {
       setLang(selected[0].id)
+      setRadioButtonValue("present")
     } else {
-      setLang(undefined)
+      setLang(null)
+      setRadioButtonValue("absent")
     }
   }
 
-  const enableLanguageSelector = () => setLanguageSelectorEnabled(true)
-
-  const disableLanguageSelector = () => {
-    setLanguageSelectorEnabled(false)
-    setLang(undefined)
+  const handleLanguageRadio = (event) => {
+    if (event.target.value === "absent") setLang(null)
+    setRadioButtonValue(event.target.value)
   }
```

Comment:
this function is called when a user clicks one of the radio buttons:

1. record the button checked in state
2. if they selected "no language", we will clear the language in state

Reply:
ditto on making it a comment for the function